src/Ninja/G1CP/Content/Misc/npc.d

```diff
@@ -219,3 +219,12 @@ func void G1CP_NpcIdSetAiVar(var int npcInstance, var string aiVarName, var int
     var C_Npc slf; slf = Hlp_GetNpc(npcInstance);
     G1CP_NpcSetAiVar(slf, aiVarName, value);
 };
+
+/*
+ * Change the value of an NPC's integer variable
+ */
+funt int G1CP_NpcChangeIntVar(var string npcName, var string npcProp, var int originalInt, var int correctedInt) {
+    var int npcId; npcId = G1CP_GetNpcInstId(npcName);
+
+    return (G1CP_ReplaceAssignInt(npcId, 0, ConcatStrings("C_Npc.", npcProp), 0, originalInt, correctedInt) > 0);
```

Comment:
Might have to double-check if the use of `ConcatStrings` causes issues here, because the result is not a constant. I do not recall if that was a prerequisite here.

Reply:
I can't test that right now, so I would be glad if you can confirm this. If you are right, this refactoring doesn't benefit the fix templates and is obsolete.

Reply:
I would say we implement this and I will have a look in February to check whether something breaks.

Reply:
I'll leave this PR open then, because I currently don't want to break anything.